src/EnergyPlus/DataHeatBalance.hh

```diff
@@ -453,6 +453,9 @@ namespace DataHeatBalance {
         Real64 ExtGrossWallArea = 0.0;                  // Exterior Wall Area for Zone (Gross)
         Real64 ExteriorTotalSurfArea = 0.0;             // Total surface area of all exterior surfaces for Zone
         int SystemZoneNodeNumber = 0;                   // This is the zone or space node number for the system for a controlled zone
+        Real64 FloorArea = 0.0;                         // Floor area used for this space
```

Comment:
Move these to be available for both zone and space under the same field names. Changes space `floorArea` to ZoneSpace `FloorArea` which cause a bunch of case changes anywhere space `floorArea` was used. Same for `totOccupants` to `TotOccupants`.